docs/src/APIs/Atmos/AtmosModel.md

```diff
@@ -26,6 +26,13 @@ ClimateMachine.BalanceLaws.source!(m::AtmosModel, source::Vars, state::Vars, dif
 ClimateMachine.BalanceLaws.init_state_prognostic!(m::AtmosModel, state::Vars, aux::Vars, localgeo, t, args...)
 ```
 
+## Compressibility
+
+```@docs
+ClimateMachine.Atmos.Compressible
+ClimateMachine.Atmos.Incompressible
```

Comment:
```suggestion
ClimateMachine.Atmos.Anelastic
```
(It's not really incompressible. But even anelastic may be going too far, because we only implement a 1d version (we do not want 3d elliptic solves...)

Reply:
Thtat is true in scampy (and I believe most single column models that make an anelastic assumption)

Reply:
@tapios how about `Anelastic1D` ? I think we already have many SingleStacks (experiments, model, ...) and this would lead to confusion.

Reply:
Sounds good.